data/Identifying_production_data_types.md

```diff
@@ -1,3 +1,216 @@
 # Data types found in production service repos
 
+---
 
+#Expression
+##Description this stores expression data of a variety of different types (microarray, RNA-Seq, Proteomics, q-PCR)
+
+##Methods
```

Comment:
Capturing what methods use what data types *today* would be useful.  Future stuff should be in another document.